nixos/tests/all-tests.nix

```diff
@@ -112,6 +112,7 @@ in {
   aesmd = runTestOn ["x86_64-linux"] ./aesmd.nix;
   agate = runTest ./web-servers/agate.nix;
   agda = handleTest ./agda.nix {};
+  age-plugin-tpm-decrypt = handleTest ./age-plugin-tpm-decrypt.nix {};
```

Comment:
Please use runTest instead of handleTest, the latter should really be deprecated by now.

Reply:
We should do a treewide and convert all tests over...

Reply:
Gotcha, curious, are there docs/PR on this change? I'm just curious to learn more.